tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es6",
     "module": "es6",
     "lib": ["es2018", "dom"],
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#--moduleresolution-bundler
Expect this would not affect end user

Reply:
why is the first letter capitalized though?

Reply:
From autocompletion. (guess this is the default)
![Screenshot 2024-06-28 at 13 32 36](https://github.com/pixiv/three-vrm/assets/26110087/a4af5eca-328d-4737-b2a9-fdef5d0ad188)

Reply:
huh.